### ISSUE TITLE
[1.18.2] Error During Sided Setup Event Phase

### ISSUE THREAD

**Issue by Thelnfamous1** [Thelnfamous1/Dungeons-Gear#206]
Discord user legospider777 reported an occasional crash during start-up.

[crash_report.txt](https://github.com/Infamous-Misadventures/Dungeons-Gear/files/10436417/crash_report.txt)
[log.txt](https://github.com/Infamous-Misadventures/Dungeons-Gear/files/10436419/log.txt)

A ConcurrentModificationException is thrown, likely indicating that ItemModelProperties is being manipulated in a thread-unsafe manner.

**Comment by Thelnfamous1** [comment]
Proposed fix: Enqueue the `init` call of `GearRangedItemModelProperties` via `FMLClientSetupEvent`.

```java 
private void doClientStuff(final FMLClientSetupEvent event) {
        event.enqueueWork(GearRangedItemModelProperties::init);
}
```

**Comment by Thelnfamous1** [comment]
Fixed in 1.19.2.

**Comment by Thelnfamous1** [comment]
1.18 is no longer supported.
